backend/handlers/event.go

```diff
@@ -43,9 +45,104 @@ func (h *EventHandler) GetMany(ctx *fiber.Ctx) error {
 }
 
 func (h *EventHandler) GetOne(ctx *fiber.Ctx) error {
-	return nil
+	eventId := ctx.Params("eventId")
+
+	context, cancel := context.WithTimeout(context.Background(), time.Duration(5*time.Second))
+	defer cancel()
+
+	event, err := h.repository.GetOne(context, eventId)
+
+	if err != nil {
+		return ctx.Status(fiber.StatusBadRequest).JSON(&fiber.Map{
+			"status":  "fail/getone/statusbadrequest",
+			"message": err.Error(),
+		})
+	}
+
+	return ctx.Status(fiber.StatusOK).JSON(&fiber.Map{
+		"status":  "success",
+		"message": "",
```

Comment:
`status` should be a number that describes the overall status of the response (e.g. 200 means OK, 401 means unauthorized,...), and `message` should give a detailed look at the response (e.g. success, DB error, Internal error,...)

Reply:
The `status` can be internal-defined, which means you can define any new status and return to front-end as long as front-end has the same status table as back-end.